api_client/python/timesketch_api_client/client.py

```diff
@@ -1245,8 +1269,8 @@ def __init__(self, timeline_id, sketch_id, api, name=None,
             timeline_id: The primary key ID of the timeline.
             sketch_id: ID of a sketch.
             api: Instance of a TimesketchApi object.
-            name: Name of the timeline (optional)
-            searchindex: The Elasticsearch index name (optional)
+            name: Name of the timelne (optional)
```

Comment:
let's not introduce a spelling mistake ;)

Reply:
ouch, yes, fixed it